### DISCUSSION TITLE
Preview release for 1.31

### DISCUSSION THREAD

**Discussion by spurin** [k3s-io/k3s#10751] [General]
Hi All,

Firstly, thanks for all of the amazing work with k3s, it's incredible!

Is there a convenient area/workflow where I can pick up a 1.31 release for early testing. I make use of k3s and other distributions of kubernetes (kubeadm) with some of my projects. I'm looking to get them all on the same page and in the interim, I'd like to start testing against a 1.31 release, even if it's a automated build, release candidate, etc.

If not, I'll look at the compilation route.

Thanks

**Comment by brandond** [comment]
You'll see an RC appear when we're ready. It is still a work in progress.

**Comment by spurin** [comment]
Thanks @brandond, best of luck with the efforts - I'll take a look at the compilation route in the interim.
